src/fetchtastic/repo_downloader.py

```
@@ -258,9 +256,9 @@ def main(config):  # log_message_func removed
                         or "y"
                     )
                     if open_folder == "y":
-                        os.startfile(
+                        os.startfile(  # type: ignore[attr-defined]
```

Ruff (0.14.10)

Starting a process without a shell

(S606)